Dockerfile.composer

```diff
@@ -0,0 +1,14 @@
+FROM composer:1.7.1
+
+ARG composer_dev_arg
+
+COPY composer.json \
+    composer.lock \
+    symfony.lock \
+    ./
+
+RUN composer --no-interaction install ${composer_dev_arg} --ignore-platform-reqs --no-autoloader --no-suggest --prefer-dist
```

Comment:
Is there value in adding `--no-scripts` ?

Reply:
Not now (since there are no scripts). We've run into problems in the past with this too (https://github.com/composer/composer/issues/7100).